src/commonTest/kotlin/com/github/quillraven/fleks/ComponentTest.kt

```diff
@@ -50,7 +50,7 @@ internal class ComponentTest {
 
     @Test
     fun addEntityToHolderWithSufficientCapacity() {
-        val entity = Entity(0)
+        val entity = Entity(0, version = 0)
```

Comment:
I suggest to use 0 as a default value in the entity constructor

Reply:
I recommend not doing that because it would make it too easy to create an invalid implementation of EntityProvider. It is now a requirement to correctly deal with version in any EntityProvider.

Reply:
We could create a method to simplify Entity creation in tests if you want?

Reply:
Nah, it is fine, no worries :)